### ISSUE TITLE
Worklog Report returning blank screen

### ISSUE THREAD

**Issue by jim-mcneill-uk** [shridhar-tl/jira-assistant#371]
### Checklist before you being

- [X] I am sure that I am already using latest version of Jira Assistant
- [X] I had verified that, no similar issues are already reported in [issue tracker](https://github.com/shridhar-tl/jira-assistant/issues)
- [X] I had tried reproducing this issue with with [Web version of Jira Assistant](https://app.jiraassistant.com) to ensure that this issue is not recently fixed and yet to be published
- [X] I had verified that, my query is not answered in [FAQ section of website](https://www.jiraassistant.com/faq)
- [X] I had verified that, my issue is not listed as [known bugs in website](https://www.jiraassistant.com/version-history)

### How do you use Jira Assistant?

Browser extension

### Are you using cloud version of Jira or self hosted (data center / server) of Jira.

Cloud Jira

### Version of Jira Assistant

2.58

### What browser are you using?

Chrome

### What OS are you using? You need not disclose this if you feel it is irr-relevant for your issue.

Windows

### Bug Description

The new worklog report is showing a completely blank screen, first error on the console is a 400, Google insight is:

Explanation

The error "400 (Bad Request)" you're seeing with your Atlassian Jira API request indicates that your JQL query is invalid. Specifically, the problem lies in how you're constructing the key in() clause within your JQL.

The Issue:

You're passing '[object Object]' as the value for the key in() clause. This suggests that instead of a proper array of issue keys, you're passing an object or an object representation as a string.

### Checklist before you submit

- [X] I have ensured not to paste any confidential information like Jira url, Mail id, etc.
- [X] I have verified my browser console logs and added necessary details (if required)
- [X] I have added required screenshots if the bug is related to UI (as necessary)

**Comment by shridhar-tl** [comment]
Somehow JQL did not form properly it seems. Please look at the console log and send me the screenshot. Additionally also look at the network tab and the params being passed to JIRA and see how query is formed. Based on the error message you provide, key is passed as stringifyed object "[object Object]". But whether that key is something you added as custom filter or it happens internally is something we need to identify. For that which exact Api call and query fails has to be identified. You network request would provide the required information.

**Comment by jim-mcneill-uk** [comment]
Do these help?
![JiraAssist-Worklog-Error-Console](https://github.com/user-attachments/assets/348cb410-2940-463c-b958-a2bfb41575d5)
![JiraAssist-Worklog-Error-Network-QueryString](https://github.com/user-attachments/assets/77ff8a80-fca1-456d-ae7d-11f7b0ec9747)
![JiraAssist-Worklog-Error-Network-QueryString2](https://github.com/user-attachments/assets/19d1b85a-8017-4241-9e89-f0fb4314eafb)

**Comment by shridhar-tl** [comment]
Yes, this helps up to an extent. Let me try to see if I was able to figure it out from code.

**Comment by jim-mcneill-uk** [comment]
I've found that it works on the webapp, sorry I should have checked that out earlier

**Comment by shridhar-tl** [comment]
The latest version v2.59 is published already and you can update it right away or wait for your turn of auto update which should happen within around 10 days.

As you have confirmed it is working with Web version, I hope it should work once you have the extension updated. Hence assuming this issue is resolved, I'm closing this ticket. Please feel free to reopen it if you still face this issue.
